op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```
@@ -0,0 +1,168 @@
+---
+## Alerts
+An incident will be declared upon receiving an alert. The metrics described below trigger various alerts with differing severities. Each alert necessitates specific actions.
+
+### faultproof-withdrawal-forgery-detected
+
+| **Network** | **Severity Level** | **Impact**                      | **Reaction**                                             | **Actions**                                                                                                                                       |
+|-------------|--------------------|---------------------------------|----------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Mainnet     | SEV1               | Fund loss                       | Immediate action required                                | - Page Faultproof (FP) team<br>- Follow critical incident procedures                                                                               |
+| Sepolia     | SEV3               | Fund loss                       | Assess acceptability of loss on Sepolia<br>Investigate the specific withdrawal | **Initial Actions:**<br>- Page FP team<br><br>**After Coordinating with FP Team:**<br>1. Blacklist the game (FP)<br>2. Execute presigned pause (Security)<br>3. Revert to permissioned game (FP) |
+
+##### Alert Description
```

Markdownlint

Expected: h4; Actual: h5
Heading levels should only increment by one level at a time

(MD001, heading-increment)